web/static/help/Messaging_es.md

```diff
@@ -0,0 +1 @@
+../../../doc/help/Messaging_es.md
```

Comment:
I renamed it cause in docs.jsx I did this:

```javascript
        if (props.site) {
            $.get(`/static/help/${props.site}_${global.window.mm_locale}.md`).then((response) => {
                this.setState({text: response});
            }, () => {
                this.setState(errorState);
            });
        } else {
            this.setState(errorState);
        }
```
So there's an easy way to show the right Messaging.md